js/samples/04.ai.a.teamsChefBot/src/index.ts

```diff
@@ -113,25 +113,19 @@ const planner = new ActionPlanner({
     defaultPrompt: 'chat',
 });
 
-// const moderator = new OpenAIModerator({
```

Comment:
argh I'm confused. Wasn't chefbot the one that uses OpenAI?

Reply:
Yup, chefbot uses OpenAI- I just removed this code because Steve commented it out and left it there. I believe the prompt management is now done by the planner